### PR TITLE
Level layout moved to LevelData

### PR DESCRIPTION
**Pull request by eon704**
## Changes:
- Moved to the `LevelData`:
  - Tilemap Walls .
  - Camera bounds.

- Multiple levels scenes are merged into Game scene.
- Trap Tilemap is painted automatically on level start.
- Cleaned up no longer used scenes.
- Cleared sorting layers on pickup prefabs.
- Other minor changes.

### CODE REVIEW

**Review by Eryngii01 — CHANGES_REQUESTED** [review]
Test Utility Tool

File: Assets/Code/Scripts/Core/Managers/GameManager.cs [hunk 1]
```diff
@@ -116,21 +124,25 @@ public class GameManager : MonoBehaviour {
         /// <remarks> Invokes the <see cref="OnSetupComplete"/> event. </remarks>
         private IEnumerator Start() {
             // Create a copy to avoid modifying source
-            LevelData sourceData = LevelManager.Instance.GetActiveLevelData();
+            LevelData sourceData = _testingLevelData != null ? _testingLevelData : LevelManager.Instance.GetActiveLevelData();
```
Comment (Eryngii01) [inline Assets/Code/Scripts/Core/Managers/GameManager.cs:127]: Thank you for including this addition! It'll greatly help cut the time taken to do extensive testing for level scenes!

File: Assets/Code/Scripts/UI/LevelsUI.cs [hunk 2]
```diff
@@ -62,11 +60,9 @@ public class LevelsUI : MonoBehaviour {
         /// </summary>
         /// <param name="levelName"> The name of the level corresponding to the LevelData name. </param>
         public void LoadLevel(string levelName) {
-            this._levelNameToLoad = levelName;
-
             this._screenWipe.gameObject.SetActive(true);
             this._screenWipe.SetRandomWipeShape();
-            this._screenWipe.WipeIn(this.LoadLevelScene);
+            this._screenWipe.WipeIn(() => this.LoadLevelScene(levelName));
```
Comment (Eryngii01) [inline Assets/Code/Scripts/UI/LevelsUI.cs:65]: Thank you for the ugly code fix 🙏

**Comment by eon704** [comment]
This update has broken the cameras in the Vertical Camera test scene. I didn't touch the scene for now, I think we can fix that I  a later PR once we try to include those features in the level design

**Review by Eryngii01 — APPROVED** [review]
LGTM!

**Pull request merged**